### ISSUE TITLE
clarification on Stock Vs Certifed firmware

### ISSUE THREAD

**Issue by Snake98bm** [iamromulan/RM520N-GL#5]
Could you clarify the difference between Stock and Certified Firmware?

**Comment by pipe2null** [comment]
Yes, some explanation would be much appreciated on the different fw's.  Thank you for dealing with Quectel and providing them here.

**Comment by Snake98bm** [comment]
I did some research, and this is what I found.
Certified firmware is what is sent to be certified by providers(Verizon,at&t, cell services providers, etc), and can be used on there networks.
Stock firmware,  is just normal firmware.
